ws-nextjs-app/pages/[service]/live/[id]/ShareButton/styles.ts

```diff
@@ -0,0 +1,42 @@
+import { Theme, css } from '@emotion/react';
+import pixelsToRem from '../../../../../../src/app/utilities/pixelsToRem';
+
+export default ({ palette, fontSizes, fontVariants, spacings, mq }: Theme) =>
+  css({
+    color: palette.BLACK,
+    ...fontSizes.pica,
+    ...fontVariants.sansBold,
+    padding: `${spacings.FULL + spacings.HALF}rem`,
+    marginTop: `${spacings.FULL}rem`,
+    marginBottom: `${spacings.TRIPLE}rem`,
+    marginInlineStart: `${spacings.DOUBLE}rem`,
+    border: `${pixelsToRem(2)}rem solid ${palette.BLACK}`,
+    backgroundColor: 'transparent',
+    cursor: 'pointer',
+    '&:hover, &:focus-visible': {
+      border: `${pixelsToRem(2)}rem solid ${palette.BRAND_BACKGROUND}`,
+      backgroundColor: palette.BRAND_BACKGROUND,
+      color: palette.WHITE,
+      span: {
+        textDecoration: 'underline',
```

Comment:
I could be wrong but i think Will was going to update the designs to not have an underline on the text off the back of a discussion in design review - maybe one to check with him ?

Reply:
I've removed the underline styling

Reply:
@HarveyPeachey we are going to need to add this back in please, otherwise you don't have any hover styling when users change colours